libs/ledger-live-common/src/families/cosmos/api/Cosmos.ts

```diff
@@ -46,7 +50,7 @@ export class CosmosAPI {
         unbondings,
         withdrawAddress,
       ] = await Promise.all([
-        this.getAccount(address),
+        this.getAccount(address, this.chainInstance.defaultPubKeyType),
```

Comment:
It doesn't seem necessary to provide the `defaultPubKeyType` to the `getAccount` function, since it already has access to it

Reply:
indeed